pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -56,6 +56,157 @@ func (l *Prometheus) GetStatus() PrometheusStatus {
 	return l.Status
 }
 
+type TopologySpreadConstraint struct {
+	// MaxSkew describes the degree to which pods may be unevenly distributed.
```

Comment:
I was torn between replicate the TopologySpreadConstraint struct or add only one property named TopologySpreadConstraintAdditionalLabels.

I'm open for suggestions

Reply:
Could we inline the original topology struct? Any special reason to replicate the whole struct?

Reply:
I tried inline but inline ends with something like that:

TopologySpreadConstraint:
    additionalLabelSelector
    TopologySpreadConstraint:
      ...properties_goes_here

Reply:
NVM! I was doing something wrong.